### PR TITLE
Add FNV-specific hash checking rule

### PR DESCRIPTION
**Pull request by Still34**
## Summary

This PR adds a new rule that detects known FNV hashes that correspond to common import names (e.g., `GetProcAddress` FNV'd ->0xf8f45725. This technique was spotted in the wild and was adopted by a Chinese APT group.

### CODE REVIEW

File: nursery/resolve-function-by-fnv-hash.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: resolve function by FNV hash
```
Comment (mr-tz) [inline nursery/resolve-function-by-fnv-hash.yml:3]: can you please add a `description:` that this is `FNV-1a`
alternatively modify the name (and rename the file)

**Review by mr-tz — CHANGES_REQUESTED** [review]
great, thanks, one small clarification request and then we can merge

File: nursery/resolve-function-by-fnv-hash.yml (hunk 1 above)
Comment (Still34) [reply nursery/resolve-function-by-fnv-hash.yml]: Done. Hope that's enough!

Comment (mr-tz) [reply nursery/resolve-function-by-fnv-hash.yml]: almost, the file needs to be renamed to `resolve-function-by-fnv-1a-hash.yml` and I think reformatted, you can test it locally using `scripts/lint.py`

Comment (Still34) [reply nursery/resolve-function-by-fnv-hash.yml]: Oh, didn't notice there was an enforced linting rule. Re-saved as LF-only and whatnot. Should pass the linting now.

Comment (Still34) [reply nursery/resolve-function-by-fnv-hash.yml]: ...wait, forgot to add the renamed file.

**Comment by mr-tz** [comment]
Awesome, thanks for the quick fixes and your contribution!

**Pull request merged**